selenium_tests/lti_emailer/emailer_base_test_case.py

```diff
@@ -0,0 +1,35 @@
+
+from selenium_tests.base_test_case import BaseSeleniumTestCase
+from selenium_tests.lti_emailer.page_objects.pin_page import LoginPage
+
+from os.path import abspath, dirname, join
+from django.conf import settings
+
+
+CANVAS_ADD_USERS = join(dirname(abspath(__file__)), 'test_data', 'roles_access.xlsx')
+CANVAS_ROLES = join(dirname(abspath(__file__)), 'test_data', 'roles.xlsx')
+
+class EmailerBaseTestCase(BaseSeleniumTestCase):
+    """
+    Bulk Create base test case, all other tests will subclass this class
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        setup values for the tests
+        """
+        super(EmailerBaseTestCase,cls).setUpClass()
+        driver = cls.driver
+        cls.USERNAME = settings.SELENIUM_CONFIG.get('selenium_username')
+        cls.PASSWORD = settings.SELENIUM_CONFIG.get('selenium_password')
+        cls.BASE_URL = '%s/courses/6389/external_tools/1759' % settings.SELENIUM_CONFIG.get('base_url')
+
+        #instantiate, then login to URL with username and password from settings, if running locally.
+        base_login = LoginPage(driver) # instantiate
```

Comment:
Python convention is two spaces before a comment that occurs on the same line as actual code (for whitespace issues see PEP-8; it's rather long but e.g. the [blank lines section](https://www.python.org/dev/peps/pep-0008/#blank-lines) and [spaces section](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements) or the [comments section](https://www.python.org/dev/peps/pep-0008/#comments)).